### PR TITLE
Updated `setup_packages()` and added `new_author()` function

### LINKED ISSUE #79: Install/update packages from get_packages() by number listed

### LINKED ISSUE #90: Create `new_author()` function
The function should append an existing description file to include new authors as they are added during the package development process.

### LINKED ISSUE #103: Add GitHub lookup if it can't find the package on CRAN for a future version of the `depends()`
Currently the `depends()` helper function loads and installs, if necessary, CRAN packages only.

### LINKED ISSUE #106: Make `setup_package ()` function accept more than two names and more than two authors
For now the function accepts only 2 names for up to 2 authors, this should be improved.  Roles of authors should also be better distinguished by the `setup_package()` function, as roles can be separate and developers can have one or many roles in the package.

### LINKED ISSUE #115: Complete README template with additional informations
Other sections can be added such as:
* How to install
* Data included
* Contributing

### LINKED ISSUE #116: Add more tests in database test templates for other common NA codings

### LINKED ISSUE #117: `get_packages` does not work according to vignette
The function appears to requires that the name of the organization as well as the name of the package are entered to work properly. This is not how the vignette or examples demonstrate.

### LINKED ISSUE #119: Update `setup_package()` so that it sets up a license.md document in new q package
With the issues with license we had on `setup_package()` and the modifications made, we forgot to make sure a license.md document is setup for documentation purposes. This should be added as one of the templates copied to new package when using the function.

### LINKED ISSUE #122: Set package logos to correct aspect ratio

### PR DESCRIPTION
**Pull request by henriquesposito**
# Description

These patch changes include updates to `setup_package()` function to accept more than two package authors and ORCID numbers and the addition of `new_author()` function that adds new package contributors to package description file. 

## Package

* Closed #122 by setting package logo to the correct aspect ratio

## Collect

* Updated `setup_package()` to make it easier to set up collaborative qPackages
  * Closed #106 by making `setup_package()` accept more than two authors as arguments
  * Closed #119 by updating `setup_package()` function so that it adds LICENSE file to new qPackages 
  * Closed #90 by adding a `new_author()` function that adds new package contributors to description file
* Closed #115 by updating README templates for new qPackages to include more sections

## Correct

* Closed #116 by adding more tests for new qPackage templates for standardizing NA formats

## Connect

* Updated `get_packages()` to make it easier to install globalgov qPackages
  * Fixed #117 by making the `get_packages()` work with package name only for qPackages part of the globalgov organization
  * Closed #79 by  updating `get_packages()` to accept listed number for package as arguments
  * Closed #103 by adding GitHub lookup for `depends()` function
* Updated user and developer vignettes to be compatible with these changes

# Checklist:

- [ ] PR form
  - [ ] I have given this pull request an informative title
  - [ ] Description above itemizes changes under subtitles, e.g. "## Collection""
  - [ ] Any closed, fixed, or related issues are referenced and explained in the description above, e.g. "Fixed #0 by adding A"
  - [ ] Package builds on my OS without issues
- [ ] PR checks all pass for latest commit
  - [ ] Package builds on Mac
  - [ ] Package builds on Windows
  - [ ] Package builds on Linux
  - [ ] CodeCov check: Package improves or maintains good test coverage
  - [ ] CodeFactor check: Package improves or maintains good style
- [ ] Documentation
  - [ ] Any new or modified functions or data have roxygen style documentation in their .R scripts
  - [ ] Any longer functions are commented inline so that it is easier to debug in the future
  - [ ] PR description above and the NEWS.md file are aligned
  - [ ] DESCRIPTION file version is bumped by the appropriate increment (major, minor, patch)

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -6,3 +6,5 @@
 ^_pkgdown\.yml$
 ^docs$
 ^pkgdown$
+^httr-oauth
+^\.httr-oauth$
```
Comment (jhollway) [inline .Rbuildignore:10]: If it's .gitignore, don't need it in .Rbuildignore? Or is it an issue for you locally?

File: DESCRIPTION [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Package: qData
 Title: An R portal for ensembled global governance data
-Version: 0.3.2
+Version: 0.3.3
```
Comment (jhollway) [inline DESCRIPTION:3]: Can you please add a `Date:` field too?

File: NEWS.md [hunk 3]
```diff
@@ -1,8 +1,24 @@
...
+* Closed #106 by making `setup_package()` accept more than two authors as arguments
+   * Closed #119 by updating `setup_package()` function so that it adds LICENSE file to new q Packages 
+* Closed #90 by adding a `new_author()` function that adds new package contributors to description file
+* Fixed #117 by making the `get_packages()` work with package name only for q Packages part of the globalgov organization
+   * Closed #79 by  updating `get_packages()` to accept listed number for package as arguments
+* Closed #103 by adding GitHub lookup for `depends()` function
+* Closed #115 by updating README templates for new q Packages to include more sections
+* Closed #116 by adding more tests for new q Package templates for standardizing NA formats
+* Closed #122 by setting package logo to the correct aspect ratio
+* Updated user and developer vignettes to be compatible with these changes
```
Comment (jhollway) [inline NEWS.md:14]: Update to reflect PR description

File: R/package_setup.R [hunk 4]
```diff
@@ -17,11 +21,18 @@
 #' @importFrom stringr str_split
 #' @examples
 #' \dontrun{
-#' setup_package("qStates", "James Hollway, Henrique Sposito")
+#' setup_package("qStates", AuthorName = c("James", "Henrique"), 
+#' AuthorSurname = c("Hollway", "Sposito"))
```
Comment (jhollway) [inline R/package_setup.R:25]: Hmm, that's a bit awkward, giving all first names and then surnames, isn't it? Maybe open a new issue to tweak this later...

File: inst/templates/qPackage-DESC2.dcf [hunk 5]
```diff
@@ -2,12 +2,14 @@ Package: {{{package}}}
...
-             family = "{{{family}}}",
-             role = c("cre", "aut", "ctb", "dtc")),
+    c(person(given = "{{{given1}}}",
+             family = "{{{family1}}}",
+             role = c("cre", "aut", "ctb", "dtc"),
+             comment = c("{{{comment1}}}")),
       person(given = "{{{given2}}}",
              family = "{{{family2}}}",
-             role = c("cre", "aut", "ctb", "dtc")))
+             role = c("cre", "aut", "ctb", "dtc"),
```
Comment (jhollway) [inline inst/templates/qPackage-DESC2.dcf:11]: So everyone gets the same roles?

File: inst/templates/qPackage-NEWS.md [hunk 6]
```diff
@@ -7,9 +7,8 @@
   * Added `Namespace` file
   * Added `News` file
```
Comment (jhollway) [inline inst/templates/qPackage-NEWS.md:8]: All capitals, no?

**Review by jhollway — CHANGES_REQUESTED** [review]

File: inst/templates/qPackage-DESC2.dcf (hunk 5 above)
Comment (henriquesposito) [reply inst/templates/qPackage-DESC2.dcf:11]: I added this to the newly open issue of arguments for `setup_packages()` This way, name sand surnames can be tweaked to became a single argument while role can be added as another argument to the `setup_package()` function.

File: .Rbuildignore (hunk 1 above)
Comment (henriquesposito) [reply .Rbuildignore:10]: This was created automatically by devtools ... would you like me to remove? Does not create an issue. I left thinking this is perhaps good practice.

**Review by jhollway — APPROVED** [review]

**Pull request merged**